src/NHSD.BuyingCatalogue.Identity.Api/Views/Shared/_CookieConsent.cshtml

```diff
@@ -0,0 +1,12 @@
+<section data-test-id="cookie-banner" class="bc-c-cookies nhsuk-header--white nhsuk-u-padding-top-3" aria-labelledby="cookies" role="banner">
+    <div class="nhsuk-width-container">
+        <h2 class="nhsuk-heading-m">Cookies on the Buying Catalogue</h2>
+        <p class="hsuk-body">We've put small files called cookies on your device to make the Buying Catalogue website work.</p>
+        <p class="hsuk-body">You can read more about our <a href="https://digital.nhs.uk/about-nhs-digital/privacy-and-cookies">cookies and privacy policy</a>.</p>
```

Comment:
Link should be redirecting to new privacy-and-cookies page `href="/privacy-policy"`

Reply:
done